jujugui/static/gui/src/app/components/mid-point/test-mid-point.js

```diff
@@ -111,7 +111,8 @@ describe('MidPoint', function() {
           <ul className="mid-point__tag-list">
             {juju.components.MidPoint.prototype.tags}
           </ul>
-          <button className="mid-point__store-button">
+          <button className="mid-point__store-button"
+            onClick={output.props.children[2].props.children[1].props.onClick}>
```

Comment:
Unfortunately because React automatically binds the click handlers we need to reach in and grab the specific handler and place it in the diff else the deepEquals will fail.